tests/run_files/segment_groups.lss

```diff
@@ -0,0 +1 @@
"/></SplitTimes><BestSegmentTime/><SegmentHistory/></Segment><Segment><Name>Reflection</Name><Icon/><SplitTimes><SplitTime name="Personal Best"/></SplitTimes><BestSegmentTime/><SegmentHistory/></Segment><Segment><Name>Rock Bottom</Name><Icon/><SplitTimes><SplitTime name="Personal Best"/></SplitTimes><BestSegmentTime/><SegmentHistory/></Segment><Segment><Name>Resolution</Name><Icon/><SplitTimes><SplitTime name="Personal Best"/></SplitTimes><BestSegmentTime/><SegmentHistory/></Segment><Segment><Name>Reflection</Name><Icon/><SplitTimes><SplitTime name="Personal Best"/></SplitTimes><BestSegmentTime/><SegmentHistory/></Segment><Segment><Name>500M</Name><Icon/><SplitTimes><SplitTime name="Personal Best"/></SplitTimes><BestSegmentTime/><SegmentHistory/></Segment><Segment><Name>1000M</Name><Icon/><SplitTimes><SplitTime name="Personal Best"/></SplitTimes><BestSegmentTime/><SegmentHistory/></Segment><Segment><Name>1500M</Name><Icon/><SplitTimes><SplitTime name="Personal Best"/></SplitTimes><BestSegmentTime/><SegmentHistory/></Segment><Segment><Name>2000M</Name><Icon/><SplitTimes><SplitTime name="Personal Best"/></SplitTimes><BestSegmentTime/><SegmentHistory/></Segment><Segment><Name>2500M</Name><Icon/><SplitTimes><SplitTime name="Personal Best"/></SplitTimes><BestSegmentTime/><SegmentHistory/></Segment><Segment><Name>3000M</Name><Icon/><SplitTimes><SplitTime name="Personal Best"/></SplitTimes><BestSegmentTime/><SegmentHistory/></Segment><Segment><Name>The Summit</Name><Icon/><SplitTimes><SplitTime name="Personal Best"/></SplitTimes><BestSegmentTime/><SegmentHistory/></Segment></Segments><SegmentGroups><SegmentGroup start="1" end="4"/><SegmentGroup start="4" end="7"/><SegmentGroup start="7" end="11"/><SegmentGroup start="11" end="15"/><SegmentGroup start="15" end="20"/><SegmentGroup start="20" end="26"/><SegmentGroup start="26" end="33"/></SegmentGroups><AutoSplitterSettings><Splits><Split>Prologue</Split><Split>Chapter1Checkpoint1</Split><Split>Chapter1Checkpoint2</Split><Split>Chapter1</Split><Split>Chapter2Checkpoint1</Split><Split>Chapter2Checkpoint2</Split><Split>Chapter2</Split><Split>Chapter3Checkpoint1</Split><Split>Chapter3Checkpoint2</Split><Split>Chapter3Checkpoint3</Split><Split>Chapter3</Split><Split>Chapter4Checkpoint1</Split><Split>Chapter4Checkpoint2</Split><Split>Chapter4Checkpoint3</Split><Split>Chapter4</Split><Split>Chapter5Checkpoint1</Split><Split>Chapter5Checkpoint2</Split><Split>Chapter5Checkpoint3</Split><Split>Chapter5Checkpoint4</Split><Split>Chapter5</Split><Split>Chapter6Checkpoint1</Split><Split>Chapter6Checkpoint2</Split><Split>Chapter6Checkpoint3</Split><Split>Chapter6Checkpoint4</Split><Split>Chapter6Checkpoint5</Split><Split>Chapter6</Split><Split>Chapter7Checkpoint1</Split><Split>Chapter7Checkpoint2</Split><Split>Chapter7Checkpoint3</Split><Split>Chapter7Checkpoint4</Split><Split>Chapter7Checkpoint5</Split><Split>Chapter7Checkpoint6</Split><Split>Chapter7</Split><Split>Epilogue</Split></Splits></AutoSplitterSettings></Run>
```

Comment:
We might need to bump the version here (and everywhere else) if custom variables land first.

Reply:
They landed first. It's now 1.8.1.